### DISCUSSION TITLE
navigating back through browser history does not restore scroll position

### DISCUSSION THREAD

**Discussion by cfklaatu** [gotson/komga#1595] [Ideas]
### Describe your suggested feature

If I navigate down my library and I click on a comic to inspect it and hit the back button when finished I would like it to back to the same spot I left off, instead of having to scroll back down through my library to find where I was. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Duplicate of #674

**Comment by Rido-o** [comment]
Any chance of this being changed? Personally for me this would be largely appreciated.

**Comment by zpaolo11x** [comment]
Any chance this or [#674](https://github.com/gotson/komga/issues/674) are addressed? For people not using pagination it can be pretty annoying to have to scroll the whole list again. Also it seems that even with pagination when going back or hitting the back button you are moved to page 1...

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.26.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.26.0))
